src/tests/validateWebhook.test.ts

```diff
@@ -0,0 +1,77 @@
+import { Nightfall } from '../nightfall'
+
+const VALID_BODY = {
+  findingsURL: 'https://nightfall.com/findings',
+  validUntil: '2021-11-06T08:05:13.193904055Z',
+  uploadID: 'MY_UPLOAD_ID',
+  findingsPresent: true,
+  requestMetadata: 'MY_SECRETS',
+  errors: []
+}
+
+const VALID_SIGNATURE = 'f3c4db6aee74079e7ba09aee16f5b729beee45edf0eb8f77173dbb479f446bda'
+
+describe('nightfall.validateWebhook()', () => {
+  it('test happy path', () => {
+    const client = new Nightfall({ webhookSigningSecret: 'MY_WEBHOOK_SECRET' })
+
+    expect(client).toBeDefined()
+    expect(typeof client.validateWebhook).toBe('function')
+
+    const isValid = client.validateWebhook(
+      VALID_BODY,
+      VALID_SIGNATURE,
+      1636099513,
+      63072000 // Two-years
```

Comment:
nit: we may want to make this even longer, just so we don't have to come back to it. 10 years? 😄